src/renderer/components/titleBar.vue

```diff
@@ -25,12 +25,14 @@
         class="frameless-titlebar-menu title-no-drag"
         @click.stop="handleMenuClick"
       >&#9776;</div>
-      <div
-        v-if="wordCount"
-        class="word-count"
-        :class="[{ 'title-no-drag': platform !== 'darwin' }]"
-        @click.stop="handleWordClick"
-      >{{ `${HASH[show]} ${wordCount[show]}` }}</div>
+      <el-tooltip class="item" effect="dark" :content="`${wordCount[show]} ${HASH[show].full + (wordCount[show] > 1 ? 's' : '')}`" :open-delay="1000" placement="bottom-end">
```

Comment:
I think we should decrease the open delay a bit.

Reply:
yeh, I set it to 500 ms.